tools/tcoffee/tcoffee.xml

```diff
@@ -0,0 +1,224 @@
+   
+    You must install T_coffee version "10.00.r1613" in your computer.
+   
+-----
+   
+    **What it does**
+
+    This tool is a wrapper for the T-Coffee multiple alignment suite. The input is a set of sequences in FASTA format that must be uploaded to Galaxy if not obtained using Galaxy tools. Apart from running on whole FASTA input, it can also run on part of input FASTA by providing list of FASTA IDs.
+   
+    This wrapper offers selected advanced T-Coffee options like the selection of the alignment methods to use: ''Pairwise Structual Method'', ''Multiple Sequence Alignment
+    Methods'' or ''Pairwise Sequence Alignment Methods''.
+   
+    The T-Coffee documentation can be found at http://www.tcoffee.org/Projects/tcoffee/documentation
+   
+-----
+
+    **Example**
+
+    Suppose you have five sequences in FASTA format:
+
+      >1aboA
+      NLFVALYDFVASGDNTLSITKGEKLRVLGYNHNGEWCEAQTKNGQGWVPS
+      NYITPVN
+     
+      >1ycsB
+      KGVIYALWDYEPQNDDELPMKEGDCMTIIHREDEDEIEWWWARLNDKEGY
+      VPRNLLGLYP
+     
+      >1pht
+      GYQYRALYDYKKEREEDIDLHLGDILTVNKGSLVALGFSDGQEARPEEIG
+      WLNGYNETTGERGDFPGTYVEYIGRKKISP
+     
+      >1vie
+      DRVRKKSGAAWQGQIVGWYCTNLTPEGYAVESEAHPGSVQIYPVAALERI
+      N
+     
+      >1ihvA
+      NFRVYYRDSRDPVWKGPAKLLWKGEGAVVIQDNSDIKVVPRRKAKIIRD
+   
+    By selecting "yes" in output fasta_aln in the wrapper, the user will obtain the multiple alignment in FASTA format:
+
+      >1aboA
+      NL-FVA---LYDFVASGDNTLSITKGEKLR-------VLGYN-------H
+      NGEWCEA--QTKN-GQGWVPSNYIT------PVN
+      >1ycsB
+      KGVIYA---LWDYEPQNDDELPMKEGDCMT-------IIHREDE-----D
+      EIEWWWA--RLND-KEGYVPRNLLG------LYP
+      >1pht
+      GYQYRA---LYDYKKEREEDIDLHLGDILTVNKGSLVALGFSDGQEARPE
+      EIGWLNGYNETTG-ERGDFPGTYVEYIGRKKISP
+      >1vie
+      DR-----------VRK--KSGAAWQGQIVGWYCTNLTPEGYAVE------
+      ------S--EAHPGSVQIYPVAALE------RIN
+      >1ihvA
+      NF-RVYYRDSRDPVWKGPA-KLLWKGEGAV-------VIQDN-------S
+      DI--------------KVVPRRKAK-----IIRD
+      
+  </help>
+  <tests>
+    <test>
+      <param name="filter_fasta" value="no"/>
+      <param name="input" value="input.fasta" ftype= "fasta"/>
+      <param name="format_output_fasta_aln" value="yes"/>
+      <output name="fasta_aln" file="output1.fasta" ftype= "fasta"/>
+    </test>
+    <test>
+      <param name="filter_fasta" value="yes"/>
+      <param name="fasta_input" value="input.fasta" ftype= "fasta"/>
+      <param name="identifiers" value="ids.txt" ftype= "txt"/>
+      <param name="format_output_fasta_aln" value="yes"/>
+      <param name="format_output_cigar" value="yes"/>
+      <output name="fasta_aln" file="output2.fasta" ftype= "fasta"/>
+      <output name="cigar_aln" file="cigar.tabular" ftype= "tabular"/>
+    </test>
+  </tests>
+  <requirements>
+    <requirement type="package" version="10.00.r1613">T-COFFEE</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" level="fatal" description="Error" />
+  </stdio>
```

Comment:
Can you add a citation?

Also, can you reorder the XML elements to be:
1. description
2. requirements
3. stdio
4. command
5. inputs
6. outputs
7. tests
8. help
9. citations